.github/workflows/static-analysis.yml

```diff
@@ -21,40 +21,11 @@ on:
 
 jobs:
   phpstan:
-    name: "PHPStan"
-    runs-on: "ubuntu-22.04"
-
-    strategy:
-      matrix:
-        php-version:
-          - "${{ inputs.php-version }}"
-
-    steps:
-      - name: "Checkout code"
-        uses: "actions/checkout@v4"
-
-      - name: "Install PHP"
-        uses: "shivammathur/setup-php@v2"
-        with:
-          coverage: "none"
-          php-version: "${{ matrix.php-version }}"
-
-      - name: "Set COMPOSER_ROOT_VERSION"
-        run: |
-          echo "COMPOSER_ROOT_VERSION=${{ inputs.composer-root-version }}" >> $GITHUB_ENV
-        if: "${{ inputs.composer-root-version }}"
-
-      - name: "Install dependencies with Composer"
-        uses: "ramsey/composer-install@v3"
-        with:
-          dependency-versions: "highest"
-          composer-options: "${{ inputs.composer-options }}"
-
-      - name: "Run a static analysis with phpstan/phpstan"
-        run: "vendor/bin/phpstan analyse -v"
+    name: "PHPStan (deprecated in favor of phpstan.yml)"
+    uses: "./.github/workflows/phpstan.yml"
```

Comment:
shouldn't this forward the inputs instead of using the defaults of the phpstan.yml workflow ?

Reply:
Oh right, good catch!